extension/src/stabilization_info.rs

```diff
@@ -11,6 +11,27 @@
 
 crate::functions_stabilized_at! {
     STABLE_FUNCTIONS
+    "1.9.0" => {
+        accessorunnest_in(cstring),
+        accessorunnest_out(accessorunnest),
+        arrow_timevector_unnest(timevector,accessorunnest),
+        rollup(timevector),
+        timevector(timestamp with time zone,double precision),
+        timevector_combine(internal,internal),
+        timevector_compound_trans(internal,timevector),
+        timevector_deserialize(bytea,internal),
+        timevector_final(internal),
+        timevector_in(cstring),
+        timevector_out(timevector),
+        timevector_serialize(internal),
+        timevector_trans(internal,timestamp with time zone,double precision),
+        unnest(timevector),
+        unnest(),
+    }
+    "1.8.0" => {
```

Comment:
Do we need these empty blocks?

Reply:
Fair question, I had thought it better to be explicit that nothing was stabilized in a particular version.  However, I'm not particular stuck on this if you feel it would be better to exclude them.